.ci/azure/linux.yml

```diff
@@ -334,7 +334,7 @@ jobs:
     displayName: 'Samples Smoke Tests'
     continueOnError: false
     condition: eq(variables['CMAKE_BUILD_SHARED_LIBS'], 'ON')
-    enabled: false
+    enabled: true
```

Comment:
why do you change it for Sample Smoke Tests?

Reply:
There were failing after align namespace merge. They were turned-off together with MO Unittests because we believed that our MO changes affected them (d89d88ecf443e54dc6b9bdbd063fb3bd87da17de)

But turns out that MO does not break it, therefore I enabled it back.